pkg/backend/display/progress.go

```diff
@@ -619,8 +622,17 @@ func (display *ProgressDisplay) printDiagnostics() bool {
 				display.println("")
 			}
 		}
+	}
 
+	// Print a link to Copilot to explain the failure.
+	// Check for SuppressPermalink ensures we don't print the link for DIY backends
+	if wroteDiagnosticHeader && !display.opts.SuppressPermalink && !display.opts.SuppressLinkToCopilot {
```

Comment:
How do we ensure we don't print this on non-cloud backends?

Reply:
Oh I guess we have `display.opts.SuppressPermalink` in that case? Nice!

Reply:
`SuppressCloudLinks` sounds good to me! The comment in the `Options` struct could be clearer too.

This PR is good to go as is though!

Reply:
`SuppressPermalink` maps to the `--suppress-permalink` flag in for `pulumi up` (you will see it if you do `pulumi help up`), and changing that would be a breaking change (and requiring a doc update). Renaming `SuppressPermalink` without renaming `--suppress-permalink` would be unhelpful.

I will update the comment.

Thanks for the review!